### ISSUE TITLE
Allow to setup windowLevel

### ISSUE THREAD

**Issue by asfdfdfd** [velikanov/SwiftPasscodeLock#25]
Because it may conflict with other libraries. For example https://github.com/terryworona/TWMessageBarManager

**Comment by ziogaschr** [comment]
This is a good idea indeed. Can you make a PR on this one?

**Comment by asfdfdfd** [comment]
I could try. But i'm not sure what is the best implementation from architectural point of view. Probably new protocol for such behaviour or maybe just parameter in PasscodeLockConfigurationType.

**Comment by ziogaschr** [comment]
I would think it fits nice in `PasscodeLockConfigurationType` :)

**Comment by ziogaschr** [comment]
What do you think of PR #28.
Adding it in `PasscodeLockConfigurationType` will not be backward compatible :)

**Comment by asfdfdfd** [comment]
1. I think that passcodeLockWindow windowLevel is not enough. Other libraries could change main windowLevel too. And to be in sync it may be better to setup main windowLevel, passcodeLockWindow behind windowLevel and passcodeLockWindow at the front windowLevel;
2. To maintain backward compatibility default protocol extension could be made. Or new configuration type probably. Not sure which is better.

**Comment by ziogaschr** [comment]
I am not so familiar with regards using many windows.
Can you make a PR please with what you think as better?

Thanks a lot for trying to make this library better :)

**Comment by asfdfdfd** [comment]
Yes, i'll do. Just have not time for it right now. Will try on next week.

**Comment by ziogaschr** [comment]
Thank you

Sent from my iPhone

On 13 ???? 2016, at 01:03, asfdfdfd <notifications@github.com<mailto:notifications@github.com>> wrote:

Yes, i'll do. Just have not time for it right now. Will try on next week.

##

**Comment by asfdfdfd** [comment]
Here it is: #37
